cartesi-rollups/api/inspect/inspect.api.mdx

```diff
@@ -26,13 +26,15 @@ import ApiTabs from "@theme/ApiTabs";
 import MimeTabs from "@theme/MimeTabs";
 import ParamsItem from "@theme/ParamsItem";
 import ResponseSamples from "@theme/ResponseSamples";
-import SchemaItem from "@theme/SchemaItem";
+import SchemaItem from "@theme/SchemaItem"
```

Comment:
Why this ; was deleted here?

Reply:
local IDE settings to remove semicolons as they are not mandatory to separate lines in JS